ms2pip/ms2pipC.py

```diff
@@ -683,48 +662,44 @@ def __init__(
         pep_file,
         spec_file=None,
         vector_file=None,
-        config_file=None,
         num_cpu=1,
         use_billiard=False,
         params=None,
         output_filename=None,
         datasetname=None,
-        return_results=True,
+        return_results=False,
```

Comment:
Maybe we should drop `return_results`, and introduce `write_results`, and always return the results. This should probably done when we refactor `MS2PIP.run`.